### PR TITLE
Implement horizontal panel layout

### PR DESCRIPTION
**Pull request by ghengeveld**
<details>
  <summary>📦 Published PR as canary version: <code>0.0.83--canary.109.797ede2.0</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @chromaui/addon-visual-tests@0.0.83--canary.109.797ede2.0
  # or 
  yarn add @chromaui/addon-visual-tests@0.0.83--canary.109.797ede2.0
  ```
</details>

### CODE REVIEW

File: .storybook/preview.tsx [hunk 1]
```diff
@@ -45,8 +45,10 @@ const Panel = styled.div<{ orientation: "right" | "bottom" }>(
     overflow: "auto",
   }),
   ({ theme }) => ({
+    containerType: "size",
+    containerName: "storybookRoot",
     position: "relative",
-    outline: `1px solid ${theme.color.border}`,
+    outline: `1px solid ${theme.appBorderColor}`,
```
Comment (MichaelArestad) [inline .storybook/preview.tsx:51]: Nice.

**Review by MichaelArestad — APPROVED** [review]
This looks good to me. Thanks for making this change!

**Review by tmeasday — COMMENTED** [review]
Seems like a decent refactoring but I don't see stories for wider viewports so I'm not sure if it actually works?

**Comment by ghengeveld** [comment]
@tmeasday I'm going to setup multi-mode snapshots in a separate PR (I tried in this one but it's more work than I'd hoped). So for now it requires manually switching to the stacked theme.

**Pull request merged**